coldfront/plugins/qumulo/tests/test_forms.py

```diff
@@ -489,3 +513,44 @@ def test_form_with_valid_data(self, mock_active_directory_api):
         }
         form = ProjectCreateForm(data=valid_data, user_id="admin")
         self.assertTrue(form.is_valid())
+
+
+class UpdateAllocationFormTests(TestCase):
+    def setUp(self):
+        build_data = build_models()
+        self.user = build_data["user"]
+        self.project1 = build_data["project"]
+        self.data = {
+            "project_pk": self.project1.id,
+            "storage_name": "TestAllocation",
+            "storage_quota": 1000,
+            "protocols": ["nfs"],
+            "storage_filesystem_path": "path_to_filesystem",
+            "storage_ticket": "ITSD-98765",
+            "storage_export_path": "/path/to/export",
+            "rw_users": ["test"],
+            "ro_users": ["test"],
+            "cost_center": "Uncle Pennybags",
+            "billing_exempt": "No",
+            "department_number": "Time Travel Services",
+            "service_rate": "consumption",
+            "billing_cycle": "monthly",
+        }
+        self.allocation = create_allocation(self.project1, self.user, self.data)
```

Comment:
creating an additional allocation *before* the allocation we're testing against will break this.  I believe this is because the allocation filter in the form isn't actually filtering down to a specific allocation
```suggestion
        create_allocation(self.project1, self.user, self.data)
        self.allocation = create_allocation(self.project1, self.user, self.data)
```